### PR TITLE
Gjoranv/metricsproxy cleanup

### PR DESCRIPTION
**Pull request by gjoranv**
FYI @freva: This removes the dual metricsproxy mode in node-admin, so it just uses the new one at port 19095.

### CODE REVIEW

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java [hunk 1]
```diff
@@ -56,7 +56,8 @@
         boolean useFdispatchByDefault();
         boolean dispatchWithProtobuf();
         boolean useAdaptiveDispatch();
-        boolean enableMetricsProxyContainer();
+        // TODO: Remove when 7.60 is the oldest model in use
+        default boolean enableMetricsProxyContainer() { return false; }
```
Comment (hmusum) [inline config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Need to wait with merging this until this criteria is fulfilled

**Review by hmusum — APPROVED** [review]

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java [hunk 2]
```diff
@@ -601,27 +600,13 @@ private void pushMetricsToContainer(NodeAgentContext context, List<DimensionMetr
...
-    }
+            String wrappedMetrics = "s:" + params.toString();
 
-    // TODO: Clean up and inline method when old metrics proxy has been discontinued.
-    private void runPushMetricsCommand(NodeAgentContext context, String wrappedMetrics, boolean newMetricsProxy) {
-        int port = newMetricsProxy ? 19095 : 19091;
-        String[] command = {"vespa-rpc-invoke",  "-t", "2",  "tcp/localhost:" + port,  "setExtraMetrics", wrappedMetrics};
-        try {
+            // Push metrics to the metrics proxy in each container
+            String[] command = {"vespa-rpc-invoke",  "-t", "2",  "tcp/localhost:19095",  "setExtraMetrics", wrappedMetrics};
```
Comment (freva) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java]: What about Vespa 6 nodes?

Comment (gjoranv) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java]: Good point, they would not get the node metrics. Let's put this on hold until stream is on 7.

**Comment by gjoranv** [comment]
@freva PTAL
Reverted "stop sending metrics to the old metricsproxy". Now only logs in DEBUG to avoid spamming the logs.

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java [hunk 3]
```diff
@@ -56,7 +56,8 @@
         boolean useFdispatchByDefault();
         boolean dispatchWithProtobuf();
         boolean useAdaptiveDispatch();
-        boolean enableMetricsProxyContainer();
+        // TODO: Remove when 7.60 is the oldest model in use
```
Comment (hmusum) [inline config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Need to update the version to 7.61 now

**Comment by gjoranv** [comment]
@freva PTAL (second last commit)

**Review by freva — APPROVED** [review]
:100:

**Review by hmusum — APPROVED** [review]
Great!

**Pull request merged**